projects/vdk-core/src/vdk/api/plugin/plugin_input.py

```diff
@@ -267,10 +266,10 @@ def ingest_payload(
             meaning all method invocations from a data job run will belong to the
             same collection.
 
-        :return: [Optional] IngestionResult, containing plugin-defined
-        information about the result from the ingestion operation. This result
-        could be used for post-ingestion processing operations like sending
-        telemetry data about the ingested payload.
+        :return: [Optional] IngestionMetadata, containing plugin-defined
+        information about the result from the ingestion operation. This
```

Comment:
information about the ingestion result, along with plugin-populated ingestion metadata?
also, maybe add an example

Reply:
Refactored to make it clearer. Thanks for the review